client/app/components/partials/FilterSettings.vue

```diff
@@ -165,6 +169,13 @@
 
       </v-flex>
 
+      <v-flex xs12 style="margin-top:10px;margin-bottom: 5px"  >
+        <v-flex id="max-homozygotes"   >
+          <v-text-field style="display:inline-block" label="Max Population Homozygotes" :rules="wholeNumRules" v-model="maxHomozygotes" hide-details>
```

Comment:
The term "Max population" has a special meaning for allele frequencies. It refers the the subpopulation with the highest allele frequency. For the homozygote counts, we are applying the filter across the grand total homozygote counts (not by subpopulation). So I think it would be clearer to use the lable "Max number of homozygotes" instead. Does that make sense?

Reply:
Yea seems reasonable to me I think it is the same number of characters as well so it should fit okay in the box --will change.

Reply:
The filter works as expected. Yay! I did notice a problem with selection. When I add a filter and a variant exists in both the custom filter and another standard filter, the selection behavior is strange. I select the variant under the custom filter, and it shows the correct variant in VariantInspectCard, but then it scrolls up and highlights the variant in the standard filter. I'm going to add this as a separate issue since it is a bug that was not introduced with this new feature.

Reply:
I noticed that too and thought it was weird behavior. I tested it with other "custom filters" and it does the same thing. I think it does that if the variant is in multiple filter categories... like if you set a custom filter and that filter has variants in it that are in other filters it does a strange... scrolling thing.